Controller/ProfileController.php

```diff
@@ -100,6 +101,15 @@ public function editAction(Request $request)
 
         return $this->render('@FOSUser/Profile/edit.html.twig', array(
             'form' => $form->createView(),
+            'data' => $this->setUserData(),
         ));
     }
+
+    /*
+     * Method to override to send data to the template
+     */
+    public function setUserData()
```

Comment:
Yes, its' a point of view. It doesn't set anything on the User entity, but it sets data for the template. I have hesitated on the naming, of course we can change the name.